pkg/model/logstore/logstore.go

```diff
@@ -422,9 +429,13 @@ func (s *LogStore) ContinuingLines(checkpoint Checkpoint) []LogLine {
 	}
 	tempLogStore.recomputeDerivedValues()
 
+	spanOptions := tempLogStore.spans
```

Comment:
i think you need to tweak the isSameSpanContinuation/isChangingSpanContinuation logic above, to only apply if you're printing the span at all

Reply:
@nicks okay I thiiiink I fixed this, can you double check my logic + test cases?